src/bp/ui-studio/src/web/components/Util/DocumentationProvider.js

```diff
@@ -23,8 +23,11 @@ class StatusBarDocumentationProvider extends Component {
 const _LinkDocumentationProvider = props => {
   const passthroughProps = _.omit(props, 'children', 'onClick', 'href', 'updateDocumentationModal')
   const onClick = e => {
-    props.updateDocumentationModal(props.file)
     e.preventDefault()
+
+    if (window.DOCS && window.DOCS[props.file]) {
+      window.open(`https://botpress.com/docs/${window.DOCS[props.file]}`, '_blank')
+    }
```

Comment:
Maybe we could put that component in ui-shared, because I think modules should also have the ability to push a link on the toolbar when you open it.

And, I also see the opportunity to bring it on the admin panel :)

Reply:
as discussed on slack, we'll have to do it once we don't have dependencies to the store actions and redux or that it can be accessed from the ui-shared folder